scalameta/tokenizers/shared/src/main/scala/scala/meta/internal/tokenizers/XmlParser.scala

```diff
@@ -123,15 +123,13 @@ class XmlParser(Block: P0,
   }
 }
 
-case class RangePosition(from: Int, to: Int)
```

Comment:
Enough programmers treat ranges as inclusive -> inclusive that I also prefer Range so I don't get surprised on Range.Exclusive vs Range.Inclusive.